CHANGELOG.md

```diff
@@ -1,8 +1,14 @@
 # Changelog
 
+## 0.4.1
+
+### Bug Fixes
+- Fix the version string to add the "v" prefix e.g v0.4.1 instead of 0.4.0 [#20](https://github.com/exoscale/vault-plugin-secrets-exoscale/pull/20)
 
 ## 0.4.0
 
+⚠️  This version has [a bug related to plugin versionning](https://github.com/exoscale/vault-plugin-secrets-exoscale/pull/20)⚠️, it is recommended to skip it and install 0.4.1 directly ⚠️
```

Comment:
I removed the binaries for the [0.4.0 release](https://github.com/exoscale/vault-plugin-secrets-exoscale/releases/tag/v0.4.0) and added a warning.